packages/calcite-components/src/components/block/block.tsx

```diff
@@ -179,6 +210,9 @@ export class Block
     connectInteractive(this);
     connectLocalized(this);
     connectMessages(this);
+    if (this.open) {
```

Comment:
Nitpick: adding a newline here would give it some breathing room between the connect utils and the `open` handling.